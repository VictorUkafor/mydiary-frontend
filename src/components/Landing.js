import React from 'react';
import man from '../assets/man1.png';
import landingLogo from '../assets/landing-logo.png';

export const Landing = () => (
  <div className="body">
    <div className="landing">
      <div className="image-ad">
        <img src={man} alt="" />
      </div>
      <div className="add">
        <img src={landingLogo} alt="" />
        <h1 className="add-h1">You&apos;ve got any idea, thought or plan!
          <br />
          Pen it down on <span className="x-h1">MyDiary</span> now
        </h1>
        <div className="d-buttons">
          <button className="direct" type="button">Have an account? Sign In</button>
          <button className="direct" type="button">You don&apos;t? Sign up today!</button>
        </div>
      </div>
    </div>
  </div>
);


export default Landing;
