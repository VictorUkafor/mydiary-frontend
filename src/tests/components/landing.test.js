import React from 'react';
import { render, cleanup } from 'react-testing-library';
import Landing from '../../components/Landing';

afterEach(cleanup);

test('<Landing/>', () => {
  const { container } = render(<Landing />);
  expect(container.firstChild).toMatchSnapshot();
});
