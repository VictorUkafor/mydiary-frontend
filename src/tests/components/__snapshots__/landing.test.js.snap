// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing/> 1`] = `
<div
  class="body"
>
  <div
    class="landing"
  >
    <div
      class="image-ad"
    >
      <img
        alt=""
        src="man1.png"
      />
    </div>
    <div
      class="add"
    >
      <img
        alt=""
        src="landing-logo.png"
      />
      <h1
        class="add-h1"
      >
        You've got any idea, thought or plan!
        <br />
        Pen it down on 
        <span
          class="x-h1"
        >
          MyDiary
        </span>
         now
      </h1>
      <div
        class="d-buttons"
      >
        <button
          class="direct"
          type="button"
        >
          Have an account? Sign In
        </button>
        <button
          class="direct"
          type="button"
        >
          You don't? Sign up today!
        </button>
      </div>
    </div>
  </div>
</div>
`;
